dashboard/app/controllers/home_controller.rb

```diff
@@ -147,6 +148,7 @@ def init_homepage
       @homepage_data[:announcement] = DCDO.get('announcement_override', nil)
       @homepage_data[:hiddenScripts] = current_user.get_hidden_script_ids
       @homepage_data[:showCensusBanner] = show_census_banner
+      @homepage_data[:showNpsSurvey] = show_nps_survey? SurveyResult::NET_PROMOTER_SCORE_2020
```

Comment:
Is this constant doing anything? Seems like it was used to store results in `SurveyResults`, but we're not storing results there any more?

Reply:
I think maybe having that constant stored in `KINDS` in SurveyResults implies there should be results for that type of survey stored there, which there won't be I don't think?